python/TestHarness/testers/RunCommand.py

```diff
@@ -20,8 +20,4 @@ def getCommand(self, options):
     return self.command
 
   def processResults(self, moose_dir, retcode, options, output):
-    reason = ''
-    if retcode != 0 :
-      reason = 'CODE %d' % retcode
-
-    return (reason, output)
+    return output
```

Comment:
I did not know what this did... so... deleted it (no longer using 'reason' to pass status)